docs/userguide/boundaryanalysis/break-rules.md

```diff
@@ -1,17 +1,33 @@
+---
+layout: default
+title: Break Rules
+permalink: /boundaryanalysis/break-rules
```

Comment:
I had mistakenly thought that it was necessary to add permalinks to each-and-every page in order to get the navigation to work properly. However, I did some testing in another branch and it seems that they are not needed at all. The site works fine without them. The only difference is that you now get `.html` appended on the end of the URLs instead of having extension-less URLs (perhaps that is cleaner maybe?).

FWIW, I think it would help to simplify things for editing for others if we don't have to maintain a URL/permalink on each page, and could instead just use the filesystem structure.

I've removed all of the permalink URLs from the markdown YAML header in every file the most recent force-push.

Reply:
I kind of like it when the pages don't end with ".html" but I see that the URLs work without that suffix, so we can choose to publish nicer links. Thanks for making it easier to edit without the permalinks!